code/lib/create-storybook/src/bin/run.ts

```diff
@@ -47,7 +48,32 @@ const createStorybookProgram = program
   .option(
     '--no-dev',
     'Complete the initialization of Storybook without launching the Storybook development server'
-  );
+  )
+  .option(
+    '--write-logs',
```

Comment:
Maybe this should be `--logfile` and accept a filepath (besides empty string for the default filename) so that it can be written elsewhere?